allure-report-preview/pom.xml

```diff
@@ -17,6 +17,8 @@
         </allure.issues.tracker.pattern>
         <allure.tests.management.pattern>http://github.com/allure-framework/allure-core/tests/%s
         </allure.tests.management.pattern>
+        <allure.stories.tracker.pattern>http://unknown.com/%s
```

Comment:
For this repo the issue tracker is the same as story tracker: https://github.com/allure-framework/allure-core/issues/%s